app/ts-cli/geminiql/parser.y

```diff
@@ -211,12 +211,32 @@ LINE_PROTOCOL:
         $$ = $1 + " " + $2
     }
 
+LINE_PROTOCOL_WITHOUT_INTO:
+    TIME_SERIE_WITHOUT_INTO
+    {
+        $$ = $1
+    }
+    |TIME_SERIE_WITHOUT_INTO TIME
+    {
+        $$ = $1 + " " + $2
+    }
+
 TIME_SERIE:
     MEASUREMENT COMMA KV_RAWS KV_RAWS
     {
         $$ = $1 + $2 + $3 + " " + $4
     }
 
+TIME_SERIE_WITHOUT_INTO:
```

Comment:
It may not be necessary to increase the `*_without_into` type. Add `|MEASUREMENT KV_RAWS` to `TIME_SERIE`.

Reply:
Yes, it's really not necessary.I have modified the code.